treeherder/model/models.py

```diff
@@ -533,3 +533,56 @@ class ReferenceDataSignatures(models.Model):
 
     class Meta:
         db_table = 'reference_data_signatures'
+
+
+@python_2_unicode_compatible
+class PerformanceFramework(models.Model):
+    name = models.SlugField(max_length=255L)
+
+    class Meta:
+        db_table = 'performance_framework'
+
+    def __str__(self):
```

Comment:
The __str__ method of a model instance should return a string representing the instance itself, not the class. So for example here you could return self.name